book/src/drivers.md

```diff
@@ -31,6 +31,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [Ember CSI](https://ember-csi.io) | `[x].ember-csi.io` | v0.2, v0.3, v1.0 | Multi-vendor CSI plugin supporting over 80 Drivers to provide block and mount storage to Container Orchestration systems. | Persistent | Read/Write Single Pod | Yes | Raw Block, Snapshot
 [GCE Persistent Disk](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver) | `pd.csi.storage.gke.io` | v0.3, v1.0 | A Container Storage Interface (CSI) Driver for Google Compute Engine Persistent Disk (GCE PD) | Persistent | Read/Write Single Pod | Yes | Snapshot
 [Google Cloud Filestore](https://github.com/kubernetes-sigs/gcp-filestore-csi-driver) | `com.google.csi.filestore` | v0.3 | A Container Storage Interface (CSI) Driver for Google Cloud Filestore | Persistent | Read/Write Multiple Pods | Yes | 
+[Google Cloud Storage](https://github.com/ofek/csi-gcs) | `storage.csi.cloud.google.com` | v1.0 | A Container Storage Interface (CSI) Driver for Google Cloud Storage | Persistent | Read/Write Multiple Pods | No |
```

Comment:
Thanks for the PR and work on the driver!

Since this driver is not maintained or supported by Google, can you replace "cloud.google.com" in the driver name with your own personal domain? And also add a note to the repo that it is not an officially supported Google product.